controller/input.lua

```diff
@@ -23,9 +23,9 @@ InputController = Class {
         end
     end;
     keypressed = function(self, key)
-        if key == "1" and world.currentRound.obstaclesPlaced < world.currentRound.maxObstacles then
```

Comment:
This line was only separate because the second part is different and it was easiest to hardcode. Your word means it can all be beautiful and generic with one if statement 😍 . I can make this change later.